README.md

```diff
@@ -9,11 +9,18 @@
   The data platform for developers.
 </p>
 
+<p align="center">
+  <a href="https://github.com/grafbase/grafbase/actions/workflows/cli-build.yml">
+    <img alt="CLI build status" src=https://github.com/grafbase/grafbase/actions/workflows/cli-build.yml/badge.svg>
```

Comment:
Should we perhaps keep this README clean and only show the badge in the README of the CLI?

Reply:
Done